### PR TITLE
Add package_releases in packagedata model

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: Tushar Goel <tushar.goel.dav@gmail.com>

Add package_releases in packagedata model to have more than one downloadable url per package

### CODE REVIEW

**Comment by TG1999** [comment]
@AyanSinhaMahapatra @pombredanne @JonoYang please have a look on these model changes, please give your suggestion on this and do let me know if these model changes are useful in anyway. We can remove all the fields from the PackageData model that will be shifted to PackageRelease after we migrate these fields to PackageRelease model.

File: src/packagedcode/models.py [hunk 1]
```diff
@@ -529,6 +529,55 @@ def update(self, other):
         return self
 
 
+wheel = "wheel"
+sdist = "sdist"
+
+RELEASE_TYPES = (
+    None,
+    wheel,
```
Comment (JonoYang) [inline src/packagedcode/models.py:537]: It feels strange to have package-specific release types in the base models.py file.

File: tests/packagedcode/data/alpine/alpine-container-layer.tar.xz-get-installed-expected.json [hunk 2]
```diff
@@ -32,7 +33,7 @@
     "code_view_url": null,
     "vcs_url": "git+http://git.alpinelinux.org/aports/commit/?id=cb70ca5c6d6db0399d2dd09189c5d57827bce5cd",
     "copyright": null,
-    "license_expression": "gpl-2.0",
+    "license_expression": "unknown",
```
Comment (JonoYang) [inline tests/packagedcode/data/alpine/alpine-container-layer.tar.xz-get-installed-expected.json:36]: The expected license expression should not be changed.

You may need to run `scancode --reindex-licenses` before regenerating the test results.

**Review by JonoYang — CHANGES_REQUESTED** [review]
@TG1999 

Where will the package_releases field be used? Is it something that scancode toolkit will populate or is it going to be used by another library?

Also, your regenerated test files have issues with some of the expected license_expressions. License expressions that were detected properly as gpl-2.0, gpl-3.0, lgpl-2.1, etc. are now being reported as `unknown`.

**Comment by pombredanne** [comment]
@TG1999 IMHO we should close this and instead have an issue to revisit the topic of packages vs. download URLs.

**Comment by TG1999** [comment]
Added issue https://github.com/nexB/scancode-toolkit/issues/3188

**Pull request closed without merging**